snabbdom/src/main/scala/snabbdom/modules/Dataset.scala

```diff
@@ -71,7 +73,7 @@ object Dataset {
       oldDataset.foreach { case (key, _) =>
         dataset.get(key) match {
           case None =>
-            if (d != null) { // TODO: does this make sense?
+            if (js.isUndefined(d)) { // TODO: does this make sense?
```

Comment:
It's here for now, we can revisit it in the future. At best it's only a micro-optimization anyway.

Reply:
I think it should be `!js.isUndefined(d)`.

Reply:
Wooops!

Reply:
```suggestion
            if (!js.isUndefined(d)) { // TODO: does this make sense?
```